mlflow/store/abstract_store.py

```diff
@@ -39,13 +40,11 @@ def create_experiment(self, name, artifact_location):
         pass
 
     @abstractmethod
-    def get_experiment(self, experiment_id, include_deleted, only_deleted):
+    def get_experiment(self, experiment_id):
         """
         Fetches the experiment from the backend store.
```

Comment:
Will this return deleted experiments?

Reply:
Yes unless permanently deletes. Left a comment here in `AbstractStore.get_experiment()` and an implementation specific on in `FileStore`